dev/DataSource/Mapping/CountryMapping.php

```diff
@@ -36,6 +36,11 @@ public static function url(): string
      */
     public static function toDataSet(Client $client, Crawler $crawler): array
     {
+        $client->waitFor('#onetrust-accept-btn-handler');
+        $cookieButton = $crawler->filterXPath(".//button[@id='onetrust-accept-btn-handler']");
+        $cookieButton->click();
+
+        $crawler = $client->refreshCrawler(); // The cookie acceptance causes a redirect, so we have to get a crawler instance for the new document
         $client->waitFor('.v-select-select');
```

Comment:
```suggestion
        $client->waitForVisibility('.v-select-select');
```
Oh, how I love JavaScript. Maybe this will help? Slightly difficult to debug this one with a headless gecko driver.